packages/sdk/chainDeploy/helpers/oracles/curveLp.ts

```diff
@@ -59,3 +56,58 @@ export const deployCurveLpOracle = async ({
 
   console.log(`Master Price Oracle updated for tokens ${underlyings.join(", ")}`);
 };
+
+export const deployCurveV2LpOracle = async ({
+  ethers,
+  getNamedAccounts,
+  deployments,
+  curveV2Pools,
+}: CurveV2LpFnParams): Promise<void> => {
+  const { deployer } = await getNamedAccounts();
+  let tx: providers.TransactionResponse;
+  let receipt: providers.TransactionReceipt;
+
+  //// CurveLpTokenPriceOracleNoRegistry
+  const cpo = await deployments.deploy("CurveV2LpTokenPriceOracleNoRegistry", {
+    from: deployer,
+    args: [],
+    log: true,
+    proxy: {
+      execute: {
+        init: {
+          methodName: "initialize",
+          args: [[], [], []],
```

Comment:
the initialize method could be used but it's not a big deal

Reply:
They will have to be used once I update this to reflect the changes on https://github.com/Midas-Protocol/contracts/pull/407/files 

But also, I wanted to avoid using the initialisation parameters so that all the pool / LP token definitions are all in one place